services/hsts/hsts.service.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const label = 'hsts preloaded'
+const { BaseJsonService } = require('..')
+
+const Joi = require('joi')
+const schema = Joi.object({
+  status: Joi.string().required(),
+}).required()
+
+module.exports = class HSTS extends BaseJsonService {
+  static get category() {
+    return 'monitoring'
+  }
+
+  static get route() {
+    return {
+      base: 'hsts',
```

Comment:
How about we use either `hsts-preload` or `hsts/preload`? As discussed in #2913, this badge isn't for hsts as much as it is for the preload list.

Reply:
I prefer `hsts/preload` :+1: